hazelcast/src/main/java/com/hazelcast/jet/impl/util/Util.java

```diff
@@ -742,4 +749,10 @@ public static MembersView getMembersView(NodeEngine nodeEngine) {
         ClusterServiceImpl clusterService = (ClusterServiceImpl) nodeEngine.getClusterService();
         return clusterService.getMembershipManager().getMembersView();
     }
+
+    public static void checkJetIsEnabled(NodeEngine nodeEngine) {
+        if (!nodeEngine.getConfig().getJetConfig().isEnabled()) {
+            throw new JetDisabledException(JET_IS_DISABLED_MESSAGE);
```

Comment:
This looks like an unregistered exception for the client exception factory, is it? see the [code](https://github.com/hazelcast/hazelcast/blob/master/hazelcast/src/main/java/com/hazelcast/jet/impl/util/ExceptionUtil.java#L91)